components/feature/media/src/test/java/mozilla/components/feature/media/fullscreen/MediaSessionFullscreenFeatureTest.kt

```diff
@@ -28,41 +36,215 @@ class MediaSessionFullscreenFeatureTest {
     val coroutinesTestRule = MainCoroutineRule()
 
     @Test
-    fun `screen orientation is updated correctly`() {
-        val mockActivity: Activity = mock()
+    fun `WHEN fullscreen is enabled then screen orientation is updated to landscape`() {
```

Comment:
Here and below: If using the GIVEN-WHEN-THEN naming scheme please capitalize these words.

Reply:
Sure. Will make the changes. Thank You!